packages/react/src/components/Breadcrumb/Breadcrumb.base.tsx

```diff
@@ -24,14 +19,13 @@ import {
   IBreadcrumbStyles,
 } from './Breadcrumb.types';
 
-/** @deprecated Use IBreadcrumbData */
-export type IBreadCrumbData = IBreadcrumbData;
-
 const getClassNames = classNamesFunction<IBreadcrumbStyleProps, IBreadcrumbStyles>();
 
-const OVERFLOW_KEY = 'overflow';
 const nullFunction = (): null => null;
 
+const OVERFLOW_KEY = 'overflow';
+const COMPONENT_NAME = 'BreadcrumbBase';
```

Comment:
I talked with @dzearing and it may be worth removing Base from component's displayNames. Some of the regex from the conformance test should also be removed to enforce that the displayName matches the components name and does not contain Base.

Reply:
Sounds good to me!